### DISCUSSION TITLE
Works with RLC-210W

### DISCUSSION THREAD

**Discussion by tcables** [thirtythreeforty/neolink#162] [General]
Hi just wanted to pop in and tell you that the RLC-210W works flawlessly with neolink out of the box on the first try. 

https://www.amazon.com/REOLINK-RLC-210W-Wireless-RLK4-210WB2-RLK4-210WB4/dp/B06XPG3ZDT

I would add in the tutorial, highlighted points about the gstreamer rtsp installation, and chmod'ing the neolink binary as executable. 

Thank you for your hard work!

**Comment by thirtythreeforty** [comment]
Hey awesome! Thanks for letting me know.
